packages/discovery-provider/plugins/pedalboard/apps/solana-relay/package.json

```diff
@@ -19,17 +19,20 @@
     "@solana/web3.js": "1.78.4",
     "axios": "1.6.8",
     "body-parser": "1.19.0",
+    "borsh": "0.3.1",
```

Comment:
borsh is on 2.0.0 but the latest version is incompatible with the way our schemas are written